<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registrationwindow</class>
 <widget class="QWidget" name="registrationwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>458</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>458</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>458</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Registrazione</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #0082b6;
</string>
  </property>
  <widget class="QLabel" name="RegistrationLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>-1</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Form di Registrazione</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>20</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="formMustBeData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>211</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;
</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldsStayAtSizeHint</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="newNameTextLabel">
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="newNameTextLineEdit">
      <property name="styleSheet">
       <string notr="true">color:black;
background-color:white;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="newSurnameTextLabel">
      <property name="styleSheet">
       <string notr="true">border:none;
</string>
      </property>
      <property name="text">
       <string>Cognome</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="newSurnameTextLineEdit">
      <property name="styleSheet">
       <string notr="true">color:black;
background-color:white;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="newUsernameTextLabel">
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="newUsernameTextLineEdit">
      <property name="styleSheet">
       <string notr="true">color:black;
background-color:white;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="newPasswordTextLineEdit">
      <property name="styleSheet">
       <string notr="true">color:black;
background-color:white;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="confirmPasswordLabel">
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="confirmPasswordTextLineEdit">
      <property name="styleSheet">
       <string notr="true">color:black;
background-color:white;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="formFacoltativeData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;</string>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="addressTextLabel">
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string>Indirizzo</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="addressTextLineEdit">
      <property name="styleSheet">
       <string notr="true">color:black;
background-color:white;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="telephonNumberTextLabel">
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string>Telefono</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="telephonNumberTextLineEdit">
      <property name="styleSheet">
       <string notr="true">color:black;
background-color:white;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="emailTextLabel">
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string>email</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="emailTextLineEdit">
      <property name="styleSheet">
       <string notr="true">color:black;
background-color:white;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="sendDataButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Registra</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;</string>
   </property>
   <property name="title">
    <string>Tipo Account</string>
   </property>
   <widget class="QRadioButton" name="BasicButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Basic</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="BusinessButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>102</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Business</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ExecutiveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>71</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Executive</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Free</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>29.90€/mese</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>9,90€/mese</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="nameMessageLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>55</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="surnameMessageLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="usernameMessageLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="passwordMessageLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="confirmPasswordMessageLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="addressMessageLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="telephonMessageLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="emailMessageLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
